public/pages/game.html

```diff
@@ -9,18 +9,15 @@
       href="./src/assets/icons/LOGO.svg"
       type="image/x-icon"
     />
-    <!--Script de Tailwind-->
+    <link rel="stylesheet" href="../css/styles.css" />
```

Comment:
I think that we shouldn't import the styles for index in this game page. Besides, styles.css no longer exists because now its name is index.css. I recommend deleting this block of code.

Reply:
Ok, that was added previously. I just leave it as it was.